src/backend/ceph.rs

```diff
@@ -599,6 +602,67 @@ impl CephBackend {
         Ok(())
     }
 
+    // get the journal path (if one exists)
+    fn get_journal_path(&self, osd_id: u64) -> BynarResult<Option<PathBuf>> {
+        //get osd metadata
+        let osd_meta = osd_metadata(&self.cluster_handle)?;
```

Comment:
Is getting the OSD metadata for a single OSD not doable? Seems wasteful to get the metadata for the entire cluster then just look for the one that matches your ID...

Reply:
hmm, I could.  I'd need to write the function.  Sure!

Reply:
hmm, change osd_metadata to take an Option<u64> probably...